src/big_integer/maximum.rs

```diff
@@ -0,0 +1,33 @@
+/// Compute the multiplication set of the three largest numbers in an array
+pub fn maximum(mut nums: Vec<i32>) -> i32 {
+    nums.sort();
+    (nums[nums.len() - 1] * nums[nums.len() - 2] * nums[nums.len() - 3])
+        .max(nums[0] * nums[nums.len() - 1] * nums[nums.len() - 2])
+        .max(nums[0] * nums[1] * nums[nums.len() - 1])
+}
```

Comment:
The complexity of this is $O(n\log n)$. I think it could be done in $O(n)$. Fell free to add some more implementations (in the same file).

Reply:
I think Under src create a array under src would be a good choice.